### ISSUE TITLE
Select current subtitle while playing delayed a bit

### ISSUE THREAD

**Issue by LeonCheung** [SubtitleEdit/subtitleedit#5496]
With "Select current subtitle while playing" toggled, and after using "Split line at cursor/video position", it won't select the next line immediately as it normally does, which would cause a tiny delay, it's tiny indeed, but it's not good when you want to set start time for the next line right after the end of the current line (when you need a gap a bit longer than the min gap), sometimes it can cause the problem like demonstrated below:

https://user-images.githubusercontent.com/494788/142355497-76cbbd63-05f3-44dc-ae9b-68ada84e3ce7.mp4

P.S.: Testing on a high-end PC, and has been noticed for years. And this distance would be even larger when working with a faster play rate. Any chance to improve it a bit? :)

**Comment by niksedk** [comment]
Is this better?
https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.3/SubtitleEditBeta.zip

**Comment by LeonCheung** [comment]
Oh I think it's much better now. 👍

**Comment by niksedk** [comment]
Thx - beta updated: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.3/SubtitleEditBeta.zip

**Comment by LeonCheung** [comment]
Hmm, it seems the issue revived again. 😂

**Comment by niksedk** [comment]
> Hmm, it seems the issue revived again. 😂

Really? Still works fine here...

**Comment by LeonCheung** [comment]
I think I figured out why. It's because I'm using the ultrafast workflow. It would work well if, say, wait for about 1 sec or longer after splitting, and then play. While in my workflow, I enabled "Set focus on mouse enter" feature, and set shortcut for "Split at cursor/video position" (say, alt + spacebar), and use Spacebar to play/pause the video, when the mouse cursor enters the waveform. So it only cost me less than 0.3 sec on all three operations. 

Can we blame an ultrafast captioner here? 🤣
